packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -215,9 +217,9 @@ export type NotificationContent = {
     }
 );
 
-export interface NotificationRequest {
+export interface NotificationRequest<T = unknown> {
```

Comment:
```suggestion
export interface NotificationRequest<T extends Record<string, unknown> = Record<string, unknown>> {
```
I'm not a fan of the `unknown` type :/